kaldb/src/main/java/com/slack/kaldb/recovery/RecoveryService.java

```diff
@@ -314,20 +314,17 @@ boolean handleRecoveryTask(RecoveryTaskMetadata recoveryTaskMetadata) {
         LOG.info(
             "Recovery task {} took {}µs, (subtask times offset validation {}, consumer prep {}, msg consumption {}, rollover {})",
             recoveryTaskMetadata,
-            TimeUnit.MICROSECONDS.convert(endTime - startTime, TimeUnit.NANOSECONDS),
-            TimeUnit.MICROSECONDS.convert(offsetsValidatedTime - startTime, TimeUnit.NANOSECONDS),
-            TimeUnit.MICROSECONDS.convert(
-                consumerPreparedTime - offsetsValidatedTime, TimeUnit.NANOSECONDS),
-            TimeUnit.MICROSECONDS.convert(
-                messagesConsumedTime - consumerPreparedTime, TimeUnit.NANOSECONDS),
-            TimeUnit.MICROSECONDS.convert(
-                rolloversCompletedTime - messagesConsumedTime, TimeUnit.NANOSECONDS));
+            nanosToMicros(endTime - startTime),
```

Comment:
I think it would be confusing to have different units for different log messages. Can we just use millis here also? Also, please update the units in the log message above for sub-task times.